### ISSUE TITLE
NameError: name 'basestring' is not defined

### ISSUE THREAD

**Issue by mno1** [digi604/django-smart-selects#63]
Hi,

I try this modul with Python 3.4.1, Django 1.6.5, django-smart-selects
1.0.9 and got following error:

```
Validating models...

Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x014CA588>
Traceback (most recent call last):
  File "C:\workspace\03-src\lib\site-packages\django\utils\autoreload.py", line 93, in wrapper
    fn(*args, **kwargs)
  File "C:\workspace\03-src\lib\site-packages\django\core\management\commands\runserver.py", line 101, in
inner_run
    self.validate(display_num_errors=True)
  File "C:\workspace\03-src\lib\site-packages\django\core\management\base.py", line 310, in validate
    num_errors = get_validation_errors(s, app)
  File "C:\workspace\03-src\lib\site-packages\django\core\management\validation.py", line 34, in get_valid
ation_errors
    for (app_name, error) in get_app_errors().items():
  File "C:\workspace\03-src\lib\site-packages\django\db\models\loading.py", line 196, in get_app_errors
    self._populate()
  File "C:\workspace\03-src\lib\site-packages\django\db\models\loading.py", line 75, in _populate
    self.load_app(app_name, True)
  File "C:\workspace\03-src\lib\site-packages\django\db\models\loading.py", line 99, in load_app
    models = import_module('%s.models' % app_name)
  File "C:\workspace\03-src\lib\importlib\__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "C:\workspace\03-src\djangoproject\service\models.py", line 34, in <module>
    class Vehicle(models.Model):
  File "C:\workspace\03-src\djangoproject\service\models.py", line 45, in Vehicle
    auto_choose=True
  File "C:\workspace\03-src\lib\site-packages\smart_selects\db_fields.py", line 20, in __init__
    if isinstance(to, basestring):
NameError: name 'basestring' is not defined
```
I found some advice: https://github.com/oxplot/fysom/issues/1

**Comment by SalahAdDin** [comment]
Hi, where i have put this code?

**Comment by elvisperez** [comment]
Hi. In all class where you have this problem

**Comment by orgkhnargh** [comment]
I am going to work on this issue today.

**Comment by SalahAdDin** [comment]
But, have i to modify theses packages files for fix this bug?

**Comment by orgkhnargh** [comment]
@SalahAdDin yes, you have to modify `django-smart-selects` package files to fix this error. However you do not have to do it yourself. @d9pouces have already did it. You may want to build a package from this repo: https://github.com/d9pouces/django-smart-selects

Django ships with built in `six` module. Take a look at this page: https://docs.djangoproject.com/en/1.7/topics/python3/

**Comment by SalahAdDin** [comment]
Ah ok, thank you, i did all manually, but, many people, included me, have the next problem: https://github.com/digi604/django-smart-selects/issues/74

**Comment by SalahAdDin** [comment]
Guys, i think that we should create a own fork of django-smart-selects and put in it all PR for update and put many features for this. SmartSelect's author don't want update this repo and this is not good.

**Comment by SalahAdDin** [comment]
Now, we have the next warning:

```
/home/tulipan/Proyectos/TiempoTurco/lib/python3.4/site-packages/smart_selects/views.py:10: RemovedInDjango19Warning: django.db.models.get_model is deprecated.
  model_class = get_model(app, model)
```

Can anyone fix it?

**Comment by jimmykobe1171** [comment]
fixed in new release 1.1.1
